api/src/contracttest/java/org/pmiops/workbench/consumer/AppsApiTest.java

```diff
@@ -78,14 +85,26 @@ static LeonardoCreateAppRequest createAppRequest() {
     request.setLabels(Map.ofEntries(entry("key1", "value1")));
     request.setDescriptorPath("descriptor/path");
     request.setDiskConfig(
-        new LeonardoPersistentDiskRequest().diskType(LeonardoDiskType.SSD).size(100));
-    request.setCustomEnvironmentVariables(new ArrayList<>());
+        new LeonardoPersistentDiskRequest()
+            .diskType(LeonardoDiskType.SSD)
+            .size(100)
+            .name("mockDisk"));
+    request.setCustomEnvironmentVariables(Map.ofEntries(entry("key1", "value1")));
     request.setExtraArgs(new ArrayList<>());
```

Comment:
nit: This seems unnecessary and confusing. Do you have to set this upon initialization to an empty list?